tests/data/reference_graphs/quantized/symmetric/densenet121.dot

```diff
@@ -1,1523 +1,1399 @@
 strict digraph  {
```

Comment:
This graph is in a pretty bad shape - the concat'ed tensor WILL contain FP32 values and the batch norm result is not quantized at all. How we should handle this "separated" batch norm situation is an open question.

The situation could be made clearer by using pre-hooks. Skip this graph in pytest as well and leave the .dot files as-is.

![Screenshot from 2020-10-13 15-10-26](https://user-images.githubusercontent.com/31695470/95859156-d9648180-0d66-11eb-85f4-58f4dc00f549.png)

Reply:
Ok, found another issue here that had better be fixed immediately - the mergeable subgraphs get merged even though there is an output edge in the middle of the subgraph, see below. The merge should not have happened at all. I will fix this separately, wait for my fix.

Reply:
Discussed with @alexsu52 - should probably make BatchNorm an INPUTS_QUANTIZABLE operation as well. The propagation mode will trigger a "wildcard" quantization for it since there is no "ScaleShift" or batch norm operation in the IR.

Reply:
#192